src/core/Text/Header/HeaderText.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useStyles } from './styles';
+import Typography from '@material-ui/core/Typography';
+
+type Type = 'bold' | undefined;
+
+interface HeaderTextProps {
+  type?: Type;
+  children: string;
+}
+
+const HeaderText: React.FC<HeaderTextProps> = ({ type, children }) => {
+  const classes = useStyles();
+
+  function getType(type: Type): string {
+    switch (type) {
+      case 'bold':
+        return classes.bold;
+      case undefined:
```

Comment:
```suggestion
      default:
```

Reply:
@S-ayanide In this case, should the variant be passes through props as default?

Reply:
Yes, if the prop passed in not present in the Variant type the switch case will handle it as default